src/workerd/api/node/node.h

```diff
@@ -97,8 +97,7 @@ void registerNodeJsCompatModules(Registry& registry, auto featureFlags) {
     // We put node:_http_server and related features behind a compat flag
     // for securing backward compatibility.
     if (isNodeHttpServerModule(module.getName())) {
-      return featureFlags.getEnableNodejsHttpServerModules() &&
-          featureFlags.getWorkerdExperimental();
+      return featureFlags.getEnableNodejsHttpServerModules();
```

Comment:
Let's separate this into a separate PR following these other edits. I'd prefer to keep it experimental just a bit longer until more of the "relevant tests" listed in `http-server-nodejs-test` are checked off, the docs PR is at least opened, and we have a running example that demonstrates this being used to enable either express or fastify, just so we can be certain the functionality is sufficient for those.

Reply:
Having express/fastify might not be related to this compat flag. Having that requirement to remove experimental would be too much.

Reply:
What I'm trying to say is: there are multiple other blockers for express that isn't related to this compat flag change.